sdk/core/src/request_options/content_range.rs

```diff
@@ -49,6 +49,14 @@ impl FromStr for ContentRange {
             })
         })?;
 
+        if cfg!(feature = "azurite_workaround") && remaining == "0--1/0" {
```

Comment:
Can we add a comment here? It would be nice to know *why* this code is necessary without having to have the associated issue in memory.